SIT.Manager.Avalonia/ViewModels/PlayPageViewModel.cs

```diff
@@ -294,6 +290,10 @@ private async Task ConnectToServer(bool launchServer = false)
 
         if (launchServer)
         {
+            if (_akiServerService.State == RunningState.Starting)
+            {
+                return;
+            }
```

Comment:
This shouldn't return, otherwise the quick play button will do nothing if the server is started. Since the state check has been fixed in the aki service we can allow it to call start and wait for the server

Reply:
Forgot to remove, will remove.